content/ADR-144.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: adr
+adr: 144
+title: Realm resolution from connection string
+date: 2022-12-13
+status: Review
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - menduz
+---
+
+## Abstract
+
+<!--
+Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the document section. **Someone should be able to read only the abstract to get the gist of what this document is about in its current state.** Abstracts should be always up to date with the current state of the document.
+-->
+
+This document describes the Realm resolution algorithms to consistently resolve servers based on a realm connection string. It resembles DNS, and uses the DAO catalysts, ENS and URLs to resolve the realms as described in [ADR-110](https://adr.decentraland.org/adr/ADR-110).
+
+## Context, Reach & Prioritization
+
+<!--
+Discuss and go into detail about the subject in question. Make sure you cover:
+- Why is this decision important
+- The urgency of the decision
+- Datapoints and related background information
+- Vocabulary and key terms
+-->
+
+Realms ([ADR-110](https://adr.decentraland.org/adr/ADR-110)) are the configurations that define what and how is executed and rendered in a Decentraland Explorer [ADR-102](https://adr.decentraland.org/adr/ADR-102). If we reason by analogy and consider an Explorer like a Web Browser, and the Scenes like WebSites. Then the Realm is the URL of that site.
+
+## Specification
+
+<!--
+The technical specification should describe the syntax and semantics of any new feature.
+-->
+
+Realm are represented as strings. Those strings need to be resolved to a `/about` endpoint to fetch the configuration.
+
+
+The mechanism to resolve that endpoint is:
+1. If the string is an URL, then it uses the URL
```

Comment:
Shouldn't it add `/about` at the end? For e.g. for worlds the URL should be `https://my-worlds-server.io/world/my-name.dcl.eth`, not `https://my-worlds-server.io/world/my-name.dcl.eth/about`